src/clusterfuzz/_internal/base/tasks.py

```diff
@@ -190,6 +195,21 @@ def get_regular_task(queue=None):
       return task
 
 
+def get_android_regular_task():
+  """Get an android regular task."""
+  if not environment.get_value('CF_ALLOWS_LISTENING_GENERIC_QUEUE') or \
```

Comment:
As discussed in meet, let's make this default as opposed to introducing new env var